src/Components/test/testassets/TestServer/Startup.cs

```diff
@@ -76,7 +77,12 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                     options.AddSupportedCultures("en-US", "fr-FR");
                     options.AddSupportedUICultures("en-US", "fr-FR");
 
-                    // Cookie culture provider is included by default.
+                    // Cookie culture provider is included by default, but we want it to be the only one.
+                    options.RequestCultureProviders.Clear();
+                    options.RequestCultureProviders.Add(new CookieRequestCultureProvider());
+
+                    // We want the default to be en-US so that the tests for bind can work consistently.
+                    options.SetDefaultCulture("en-US"); 
```

Comment:
Does the fact that we use the cookie culture provider mean that, if you've set a non-en-US culture, then the tests will be broken on your machine until you clear that cookie?

I don't have a great alternative suggestion. Maybe the tests that are meant to *only* work in en-US could take care of forcing the thread to en-US in setup and reverting to the previous culture in teardown.

Maybe it's not really a problem. If this fixes things in default cases I'm fine with proceeding as-is, at least until any actual issue becomes evident.

Reply:
I don't think the cookies in selenium are durable across tests/runs. We already have tests that flip this setting back and forth between `en-US` and `fr-FR` and it hasn't caused any problems.